runtime/cpp/include/bpf-api.h

```diff
@@ -55,6 +56,8 @@ class bpf_buffer {
     virtual ~bpf_buffer() = default;
 };
 
+using bpf_map_store = std::unordered_map<int, bpf_map*>;
```

Comment:
I think maybe it's better to embed and manager bpf_map_store in wasm_bpf_program?

Reply:
If map fds are cached in wasm_bpf_program, we have to iterate over all programs a wasm program opened when we query bpf_map* by fd integers

Reply:
Fixed. now we use `bpf_map_get_info_by_fd` instead of caching map fds.